edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -320,19 +322,19 @@ static string GetLocalConfigFilePath(IConfiguration configuration, ILogger logge
             return localConfigPath;
         }
 
-        static Option<ulong> GetStorageMaxTotalWalSizeIfExists(IConfiguration configuration)
+        static Option<T> GetStorageConfigIfExists<T>(string fieldName, IConfiguration configuration)
```

Comment:
why do you need to make it with type T?  Just return Option<ulong>

Reply:
It's used for multiple types. One is an enum, the other is ulong